middle_end/flambda2/term_basics/alloc_mode.mli

```diff
@@ -12,26 +12,28 @@
 (*                                                                        *)
 (**************************************************************************)
 
-type t = private
-  | Heap  (** Normal allocation on the OCaml heap. *)
-  | Local  (** Allocation on the local allocation stack. *)
+module For_types : sig
+  type t = private
+    | Heap  (** Normal allocation on the OCaml heap. *)
+    | Local  (** Allocation on the local allocation stack. *)
+    | Heap_or_local  (** Allocation with unknown location *)
```

Comment:
I think it might be worth adding a comment to point out that `Local` means something different from in `Lambda`.

Reply:
(and different from `For_allocations`, indeed)